data/102/2612/en_US.json

```diff
@@ -3,9 +3,9 @@
 		"2612": {
 			"name": "WRONG_MAIL_ADDRESS",
 			"message": "The e-mail address you entered is not\ncorrect.\n\nPlease check the e-mail address\nyou entered and try again.",
-			"short_description": "Unknown cause.",
+			"short_description": "The entered e-mail address is incorrect.",
```

Comment:
Just like with the short solutions, if you're going to add a short description then we also ask that you add a long description. This, too, supports markdown and is intended for the website